deep-linking/README.md

```diff
@@ -0,0 +1,71 @@
+# bpmn-js collapses subprocesses example
+
+This example shows how to use the [rootElements API](https://github.com/bpmn-io/diagram-js/blob/master/lib/core/Canvas.js) of [bpmn-js](https://github.com/bpmn-io/bpmn-js) to switch between different diagram layers.
+
+![Screenshot](resources/screenshot.png)
```

Comment:
Is the padding-left (40px) by design for the breadcrumbns? I did stuble upon it in this example:

![image](https://user-images.githubusercontent.com/58601/154996937-b04397bb-fa60-41b1-8483-9779af64a3a7.png)

Reply:
Expected behavior: In the viewer the breadcrumbs start top-left (no padding).

Reply:
Let us make it "left padding == top padding or something, visually speaking.

![image](https://user-images.githubusercontent.com/58601/155007470-8697637f-293a-4d04-998e-ea396b49ccd7.png)

Reply:
cf. https://github.com/bpmn-io/bpmn-js/pull/1608

I decided to give position it in a way that the modeler only moves the breadcrumbs right but not down. It should work for most viewer integrations, where a Diagram is displayed in some sort of border.

For this PR, we can make it flush with the heading with custom CSS instead.